### DISCUSSION TITLE
Is there any way to speed up?

### DISCUSSION THREAD

**Discussion by yjtwinpark** [Kitware/trame#214] [Q&A]
I made example playing animation by openning vtm files (totally 6GB, 30 time step, 200mb for each step), it is too slow to play.

In fact, if i read just one time step, and pan or rotate screen, speed is good to use,

but, if press play button and play for time, (like as TimeAnimation Example, can.ex) too slow to use.

Even I reduce file size to 1GB(remove some result variable), stiil too slow.

Is there any way to speed up? (such as, open at one for timestep as one timestep's all block?)

**Comment by jourdain** [comment]
Yes many options are possible. But they will depend on your data and exact needs. Without data, code or other details, it is hard to say. If you feel like it, [Kitware can work with you](https://www.kitware.com/contact/) on getting your problem solved.
